### PR TITLE
fix(input): `v-model` modifiers not work

### PR DESCRIPTION
**Pull request by John60676**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

fix this issue: #3816 

reference: https://github.com/vuejs/vue/issues/5847

### CODE REVIEW

File: components/input/Input.jsx [hunk 1]
```diff
@@ -115,6 +115,10 @@ export default {
       }
     },
     onChange(e) {
+      // fix this isssue: https://github.com/vueComponent/ant-design-vue/issues/3816
+      // reference: https://github.com/vuejs/vue/issues/5847 and https://github.com/vuejs/vue/issues/8431
+      this.$forceUpdate();
```
Comment (tangjinzhou) [inline components/input/Input.jsx:120]: 会导致 v-model.trim 时 无法输入  中间带空格的字符吧

Comment (John60676) [reply components/input/Input.jsx:120]: 哦，也对，那我改改

Comment (John60676) [reply components/input/Input.jsx]: done

Comment (yinlei1126) [reply components/input/Input.jsx]: 可以改成如 原生一样吗 v-model="", v-model.trim 为什么一定要加一个 v-model:value, 这样感觉不理想
element ui 里的 el-input 组件，是支持 v-model 方式的，如 原生一样，希望能调整下

**Pull request merged**